torch/_export/serde/serialize.py

```diff
@@ -1335,13 +1335,20 @@ def serialize_graph(self, graph_module: torch.fx.GraphModule) -> Graph:
             is_single_tensor_return=self.graph_state.is_single_tensor_return,
         )
 
+    def serialize_graph_module_metadata(self, meta: Dict[str, str]):
+        ret = {}
+        if custom := meta.get("custom"):
```

Comment:
what about other meta fields? 

Since meta is a Dict[str, str], all fields can be serialized.

Reply:
I believe meta is not Dict[str, str] and I type it wrong...